cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -326,6 +333,17 @@ impl YaccParser {
         }
     }
 
+    fn parse_action(&mut self, i: usize) -> YaccResult<(usize, String)> {
+        match RE_ACTION.find(&self.src[i..]) {
+            Some(m) => {
+                assert!(m.start() == 0 && m.end() > 0);
+                let s = self.src[i + 1..i + m.end() - 1].trim().to_string();
+                Ok((i + m.end(), s))
+            }
+            None => Err(self.mk_error(YaccParserErrorKind::IllegalString, i))
```

Comment:
I think you'll need a new error kind here, something like `IncompleteAction` or `IncompleteAction`.